ui.content/src/main/content/jcr_root/content/experience-fragments/hclecomm/language-masters/en/site/simple-experience-fragment/master/.content.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Rajamouli-hcl : we dont need to checkin the exp fragment content as part of codebase

Reply:
changes updated as per your review